canonical/ubuntu-pro-client
# ci: create pr and release checklist comment actions

## Why is this needed?

This revamps our PR checks
- moves checklists out of PR description into bot-maintained comment
- fails action if bug-refs aren't confirmed or checklist items aren't completed
- moves translation template update requirement to release time

There is an extra commit here to give the action some sample data and to prevent other actions from running on this PR. ***We must remove that commit before merging.***

## Test Steps

Play with the checklist comment checkboxes and use the checkbox at the bottom of this description to re-run the actions. See how if the checklist is not properly filled out then the action fails.

---
